File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.py
def solution(left, right):
    result=0
    for item in range(left,right+1):
        total=0
        for k in range(1,item+1):
            if item%k==0:
                total+=1
        if total%2==0:
            result+=item
        else:
            result-=item
    return result

File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.js
function solution(left, right) {
    let result=0;
    for(let i=left;i<=right;i++){
        if(Math.sqrt(i)==~~(Math.sqrt(i))){
            result-=i;
        }
        else
            result+=i;
        console.log(result)
    }
    return result;
}